### PR TITLE
Mark changes on PPL version in pink

### PR DESCRIPTION
**Pull request by lvnhmd**

### CODE REVIEW

File: assets/scss/projects.scss [hunk 1]
```diff
@@ -269,6 +269,10 @@ $light-green: #26A197;
     &.comments {
       background: $light-green;
     }
+
+    &.changed {
+      background: #ff1493;
```
Comment (joefitter) [inline assets/scss/projects.scss]: could we add a variable please?

File: client/components/application-summary.js [hunk 2]
```diff
@@ -95,6 +97,24 @@ class ApplicationSummary extends React.Component {
     }
   }
 
+  changed = (key) => {
+    const fields = this.props.fieldsBySection[key] || [];
+    if(this.props.changed.includes(key) || this.props.changed.some(k=> fields.includes(k) )) {
+      return CHANGED;
+    }
+  }
```
Comment (joefitter) [inline client/components/application-summary.js]: @lvnhmd I think you'll also need to include looping sections - see the use of `repeats` property in the comments code

File: assets/scss/projects.scss (hunk 1 above)
Comment (lvnhmd) [reply assets/scss/projects.scss]: done

File: client/components/application-summary.js (hunk 2 above)
Comment (lvnhmd) [reply client/components/application-summary.js]: done

**Comment by joefitter** [comment]
I have boueuouf (:cow2:) with adding 2 no-op reducers for the same thing, more so with the naming of them. Could we have a single reducer, called `amends`. Also this only shows amends on first page load, we'll probably want to show live changes as a user is amending fields

File: client/components/application-summary.js [hunk 3]
```diff
@@ -95,6 +98,27 @@ class ApplicationSummary extends React.Component {
     }
   }
 
+  changed = (key, subsection) => {
+    const fields = this.props.fieldsBySection[key] || [];
+    if(this.props.latest.includes(key) || this.props.latest.some(k=> fields.includes(k)) || this.props.latest.includes(subsection.repeats)) {
```
Comment (joefitter) [inline client/components/application-summary.js]: `if (`
`k =>`
spaces please - not really your fault - this should be picked up by lint - I guess it's ignoring JSX

Comment (joefitter) [reply client/components/application-summary.js]: actually that's a lie - this isn't a jsx file - our linting is obvs borked

**Pull request merged**